server/services/licenceService.ts

```diff
@@ -263,7 +263,8 @@ export default class LicenceService {
     user?: User,
   ): Promise<LicenceSummary | null | undefined> {
     const licences = await this.licenceApiClient.matchLicences(statuses, null, null, nomisIds, null, null, null, user)
-    return _.head(licences)
+    const licencesSortedDesc = licences.sort((a, b) => a.licenceId - b.licenceId)
+    return _.last(licencesSortedDesc)
```

Comment:
sort licences by licenceId and return the latest licence (last of the sorted array). Just decided to edit this service method so it does what it is meant to when multiple licences are present.